e2e/e2e.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BLUE='\033[0;36m'
+PURPLE='\033[0;35m'
+NC='\033[0m' # No Color
+
+#YOU NEED TO BE INSIDE A K8S CLUSTER TO RUN THIS#
+
+NS=$(cat /dev/urandom | tr -cd 'a-f0-9' | head -c 12) #generate random namespace
+ROOT=$(git rev-parse --show-toplevel)   #get root of git repo
+QUOTACLAIM='resourcequotaclaims.cagip.github.com'
+
+#clean up function. If you want to totaly remove kotary after test, uncomment the lines
+CleanUp () {
+    echo -e "\\n${BLUE}Starting CleanUp ${NC}\\n"
+    kubectl delete ns $NS
+    #kubectl delete configmap -n kube-system kotary-config
+    #kubectl delete deployment -n kube-system kotary
+    #kubectl delete crd resourcequotaclaims.cagip.github.com
+}
+
+echo -e "${BLUE}====== Starting SetUp ======${NC} \\n"
+
+if ! kubectl apply -f $ROOT/artifacts/crd.yml ;
+ then echo -e "\\n${RED}CONNECT TO A CLUSTER BEFORE RUNNING THIS EXECUTABLE${NC}\\n" && exit 1 ; fi
+
+kubectl apply -f $ROOT/artifacts/deployment.yml
+kubectl apply -f $ROOT/e2e/KotaryService/ConfigMap.yaml -n kube-system
+
+kubectl create ns $NS
+while ! kubectl get pods -n kube-system | grep kotary | grep Running > /dev/null ; do echo -e "${BLUE}.... Waiting for Kotary pod to be Running ....${NC}" ; sleep 2; done
```

Comment:
nit: You could use a different command to _ensure_ 1/1 are running, but this is fine for me.